iidm/iidm-api/src/main/java/com/powsybl/iidm/network/VoltageLevel.java

```diff
@@ -521,6 +522,34 @@ default void removeInternalConnections(int node1, int node2) {
             throw new UnsupportedOperationException();
         }
 
+        /**
+         * Get the switches connected to the {@code node}.
+         *
+         * @throws com.powsybl.commons.PowsyblException if node is not found.
+         */
+        Stream<Switch> getSwitchStream(int node);
+
+        /**
+         * Get the switches connected to the {@code node}.
+         *
+         * @throws com.powsybl.commons.PowsyblException if node is not found.
+         */
+        Iterable<Switch> getSwitches(int node);
+
+        /**
+         * Get the indices of the nodes connected with an internal connections to the {@code node}.
+         *
+         * @throws com.powsybl.commons.PowsyblException if node is not found.
+         */
+        IntStream getNodeStreamInternalConnectedTo(int node);
+
+        /**
+         * Get the internal connections connected to the {@code node}.
+         *
+         * @throws com.powsybl.commons.PowsyblException if node is not found.
+         */
+        Iterable<Integer> getNodesInternalConnectedTo(int node);
```

Comment:
I known that the API is everywhere made of Stream + Iterable to let impl optimize/lazy load data, but here the number of object returned is very small and maybe we can just return a List instead of the Iterable?

Reply:
Done.